File: Frontend/src/app/Core/service/model/RestaurantNearByRequest.ts
import {Page} from "./page";

export class RestaurantNearByRequest{
  page : Page;
  latitude : string | null;
  longitude : string | null;

  constructor() {
    this.page = new Page();
    this.latitude = '21.0492744';
    this.longitude = '105.7324997';
  }

}

File: Backend/foodee_service/src/main/java/org/example/foodee_service/dto/request/restaurant_request/RestaurantNearByRequest.java
package org.example.foodee_service.dto.request.restaurant_request;

import lombok.*;
import lombok.experimental.SuperBuilder;
import org.example.foodee_service.dto.request.common.PagingQueryRequest;


@EqualsAndHashCode(callSuper = true)
@Data
@AllArgsConstructor
@NoArgsConstructor
@SuperBuilder
public class RestaurantNearByRequest extends PagingQueryRequest {
    Double latitude;
    Double longitude;
}
